console.ts

```diff
@@ -0,0 +1,82 @@
+class ConsoleContext {
```

Comment:
why not
`type ConsoleContext = Set<any>;`

Reply:
Not quite sure if there would be any more fields added to the ConsoleContext, so I left it that way so could later introduce new props if necessary. Could change to type though